context/app/static/js/components/detailPage/Attribution/Attribution.tsx

```diff
@@ -1,32 +1,95 @@
+import SummaryPaper from 'js/shared-styles/sections/SectionPaper';
+import LabelledSectionText from 'js/shared-styles/sections/LabelledSectionText';
+import ContactUsLink from 'js/shared-styles/Links/ContactUsLink';
+import InfoAlert from 'js/shared-styles/alerts/InfoAlert';
 
 function Attribution() {
   const {
-    entity: { group_name, created_by_user_displayname, created_by_user_email, entity_type },
+    entity: {
+      group_name,
+      created_by_user_displayname,
+      created_by_user_email,
+      entity_type,
+      processing,
+      creation_action,
+      descendants,
+    },
   } = useFlaskDataContext();
 
-  return (
-    <DetailPageSection id="attribution">
-      <SectionHeader iconTooltipText={`Information about the group registering this ${entity_type?.toLowerCase()}.`}>
-        Attribution
-      </SectionHeader>
-      <FlexPaper>
-        <SectionItem label="Group">{group_name}</SectionItem>
-        <SectionItem label="Registered by" ml>
+  const isProcessedDataset = entity_type === 'Dataset' && processing === 'processed';
+  const isVisLiftedDataset = descendants?.find((descendant) => descendant.dataset_type === 'Histology [Image Pyramid]');
+  const isHiveProcessedDataset = isProcessedDataset && creation_action === 'Central Process';
+  const isSupportDataset = entity_type === 'Support';
+
+  const showContactAndAlert = isHiveProcessedDataset ?? isSupportDataset;
+  const showRegisteredBy = !(isProcessedDataset ?? isVisLiftedDataset ?? isHiveProcessedDataset ?? isSupportDataset);
+
+  const tooltips = {
+    group: 'This is the group that provided the raw dataset.',
+    contact: 'This is the contact for this data.',
+  };
+
+  const hiveInfoAlertText = `The data provided by the ${group_name} Group was centrally processed by HuBMAP. The results of this processing are independent of analyses conducted by the data providers or third parties.`;
+
+  const sections: {
+    label: string;
+    text: React.ReactNode;
+    tooltip?: string;
+  }[] = [
+    {
+      label: 'Group',
+      text: group_name,
+      tooltip: showRegisteredBy ? undefined : tooltips.group,
+    },
+  ];
+
+  if (showContactAndAlert) {
+    sections.push({
+      label: 'Contact',
+      text: <ContactUsLink> HuBMAP Help Desk </ContactUsLink>,
+      tooltip: tooltips.contact,
+    });
+  } else if (showRegisteredBy) {
+    sections.push({
+      label: 'Registered by',
+      text: (
+        <Stack spacing={1}>
           {created_by_user_displayname}
           <EmailIconLink email={encodeURI(created_by_user_email)} iconFontSize="1.1rem">
             {created_by_user_email}
           </EmailIconLink>
-        </SectionItem>
-      </FlexPaper>
+        </Stack>
+      ),
+    });
+  }
+
+  return (
+    <DetailPageSection id="attribution">
+      <SectionHeader
+        iconTooltipText={
+          showRegisteredBy ? `Information about the group registering this ${entity_type?.toLowerCase()}.` : undefined
+        }
```

Comment:
This may be more of a personal preference - in cases like this I've been favoring defining the prop in the function body to avoid long lines in the JSX markup:

```ts
const iconTooltipText = showRegisteredBy ? `Information about the group registering this ${entity_type?.toLowerCase()}.` : undefined;
```

```tsx
<SectionHeader iconTooltipText={iconTooltipText}>
```

Reply:
I like that!